addons/payment/views/payment_templates.xml

```diff
@@ -373,8 +377,6 @@
 
     <!-- Verified token checkmark -->
     <template id="verified_token_checkmark" name="Payment Verified Token Checkmark">
-        <t t-if="0" name="payment_demo_hook"/>
-        <t t-else="">
```

Comment:
Since the next bloc ins't in the `if-else` anymore, you can fix the indentation =]